tests/testthat/test_list.R

```diff
@@ -26,3 +26,16 @@ test_that("one-level lists can be read", {
   res <- dbGetQuery(con, "SELECT ['Hello', 'World'] a union all select ['There']")$a
   expect_equal(res, list(c("Hello", "World"), c("There")))
 })
+
+test_that("rel_filter() handles LIST logical type", {
+  con <- dbConnect(duckdb())
+  on.exit(dbDisconnect(con, shutdown = TRUE))
+
+  df1 <- tibble::tibble(a = list(1, c(1,2)))
+
+  rel1 <- rel_from_df(con, df1)
+  rel2 <- rel_filter(rel1, list(expr_constant(TRUE)))
```

Comment:
Thanks, sorry for the confusion. I'm not sure we even need the filter here for the roundtrip, but it doesn't hurt.